lib/ui/profile/UserProfileListItem.dart

```diff
@@ -0,0 +1,157 @@
+
+  final EdgeInsets edgePadding;
+  final EdgeInsets simpleEdgePadding;
+
+  final double leadingWidth;
+  final double leadingIconHeight;
+  final double trailingWidth;
+  final double trailingIconHeight;
+
+  const UserProfileListItemStyle({
+    this.titleTextStyle,
+    this.destructiveTextStyle,
+    this.edgePadding,
+    this.simpleEdgePadding,
+    this.leadingWidth,
+    this.leadingIconHeight,
+    this.trailingWidth,
+    this.trailingIconHeight,
+  });
+
+  UserProfileListItemStyle copyWith({
+    TextStyle titleTextStyle,
+    TextStyle destructiveTextStyle,
+    EdgeInsets edgePadding,
+    EdgeInsets simpleEdgePadding,
+    double leadingWidth,
+    double leadingIconHeight,
+    double trailingWidth,
+    double trailingIconHeight,
+  }) {
+    return UserProfileListItemStyle(
+      titleTextStyle: titleTextStyle ?? this.titleTextStyle,
+      destructiveTextStyle: destructiveTextStyle ?? this.destructiveTextStyle,
+      edgePadding: edgePadding ?? this.edgePadding,
+      simpleEdgePadding: simpleEdgePadding ?? this.simpleEdgePadding,
+      leadingWidth: leadingWidth ?? this.leadingWidth,
+      leadingIconHeight: leadingIconHeight ?? this.leadingIconHeight,
+      trailingWidth: trailingWidth ?? this.trailingWidth,
+      trailingIconHeight: trailingIconHeight ?? this.trailingIconHeight,
+    );
+  }
+}
+
+class _DefaultStyle extends UserProfileListItemStyle {
+  final TextStyle titleTextStyle = const TextStyle(
+    fontWeight: FontWeight.normal,
+    fontSize: 17,
+  );
+
+  final TextStyle destructiveTextStyle = const TextStyle(
+    fontWeight: FontWeight.normal,
+    color: Color(0xffff6060),
+    fontSize: 17,
+  );
+
+  final EdgeInsets edgePadding = const EdgeInsets.only(
+    top: 10.8,
+    bottom: 10.8,
+    left: 12.5,
+    right: 31.5,
+  );
+
+  final EdgeInsets simpleEdgePadding = const EdgeInsets.only(
+    top: 10.8,
+    bottom: 10.8,
+    left: 33.5,
+    right: 31.5,
+  );
+
+  final double leadingWidth = 41;
+  final double leadingIconHeight = 20;
+  final double trailingWidth = 7.5;
+  final double trailingIconHeight = 13;
+
+  const _DefaultStyle({
+    TextStyle titleTextStyle,
+    TextStyle destructiveTextStyle,
+    EdgeInsets edgePadding,
+    EdgeInsets simpleEdgePadding,
+    double leadingWidth,
+    double leadingIconHeight,
+    double trailingWidth,
+    double trailingIconHeight,
+  });
+}
+
+const UserProfileListItemStyle _defaultStyle = const _DefaultStyle();
+
+class UserProfileListItem extends StatelessWidget {
+  final UserProfileListItemViewModel _viewModel;
+  final UserProfileListItemStyle _style;
+
+  const UserProfileListItem({
+    Key key,
+    UserProfileListItemViewModel viewModel,
+    UserProfileListItemStyle style = _defaultStyle,
+  })  : this._viewModel = viewModel,
+        this._style = style,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
+      onTap: () => _viewModel.onTap(_viewModel.title),
```

Comment:
why pass title?

Reply:
Is not necessary. I applied just to print the selected element.